rosmon_core/src/launch/launch_config.cpp

```diff
@@ -381,9 +386,9 @@ void LaunchConfig::parseNode(TiXmlElement* element, ParseContext ctx)
 	if(!fullNamespace.empty())
 		node->setNamespace(fullNamespace);
 
-	if(respawn)
+        if(m_respawnAll || respawn)
 	{
-		node->setRespawn(ctx.parseBool(respawn, element->Row()));
+		node->setRespawn(m_respawnAll || ctx.parseBool(respawn, element->Row()));
```

Comment:
force_false should override respawn to false. I suggest keeping two parameters `m_respawnObey` and `m_respawnAll`.
```
if (m_respawnObey) {
        node->setRespawn(ctx.parseBool(respawn, element->Row()));
} else {
         node->setRespawn(m_respawnAll);
}
```